modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/UserCardTag.java

```diff
@@ -0,0 +1,512 @@
+			jspWriter.write("type=\"checkbox\">");
+			jspWriter.write("<span class=\"custom-control-label\"></span>");
+		}
+
+		jspWriter.write("<div class=\"aspect-ratio-item-center-middle ");
+		jspWriter.write("card-type-asset-icon\">");
+		jspWriter.write("<span class=\"sticker sticker-user-icon ");
+
+		if (Validator.isNotNull(_userColorClass)) {
+			jspWriter.write("sticker-" + _userColorClass + " ");
+		}
+
+		jspWriter.write("sticker-circle\">");
+		jspWriter.write("<span class=\"sticker-overlay\">");
+
+		if (Validator.isNotNull(_imageSrc)) {
+			jspWriter.write("<img alt=\"thumbnail\" ");
+			jspWriter.write("class=\"aspect-ratio-item-fluid\" src=\"");
+			jspWriter.write(_imageSrc);
+			jspWriter.write("\"	/>");
+		}
+		else {
+			jspWriter.write("<svg class=\"lexicon-icon lexicon-icon-user\" ");
+			jspWriter.write("role=\"presentation\" viewBox=\"0 0 512 512\">");
+			jspWriter.write("<use xlink:href=\"");
+			jspWriter.write(_spritemap);
+			jspWriter.write("#user\" /></svg>");
+		}
+
+		jspWriter.write("</span>");
+		jspWriter.write("</span>");
+		jspWriter.write("</div>");
+
+		if (_selectable) {
+			jspWriter.write("</label>");
+			jspWriter.write("</div>");
+		}
+
+		jspWriter.write("</div>");
+
+		jspWriter.write("<div class=\"card-body\">");
+		jspWriter.write("<div class=\"card-row\">");
+		jspWriter.write("<div class=\"autofit-col autofit-col-expand\">");
+		jspWriter.write("<p class=\"card-title\">");
+		jspWriter.write("<span class=\"text-truncate-inline\">");
+
+		if (!_disabled && Validator.isNotNull(_href)) {
+			jspWriter.write("<a class=\"text-truncate\" href=\"");
+			jspWriter.write(_href);
+			jspWriter.write("\">");
+			jspWriter.write(_name);
+			jspWriter.write("</a>");
+		}
+		else {
+			jspWriter.write("<span class=\"text-truncate\">");
+			jspWriter.write(_name);
+			jspWriter.write("</span>");
+		}
+
+		jspWriter.write("</span>");
+		jspWriter.write("</p>");
+		jspWriter.write("<p class=\"card-subtitle\">");
+		jspWriter.write("<span class=\"text-truncate-inline\">");
+		jspWriter.write("<span class=\"text-truncate\">");
+		jspWriter.write(_subtitle);
+		jspWriter.write("</span>");
+		jspWriter.write("</span>");
+		jspWriter.write("</p>");
+
+		if (!ListUtil.isEmpty(_labels)) {
+			jspWriter.write("<div class=\"card-detail\">");
+
+			for (LabelItem labelItem : _labels) {
+				jspWriter.write("<span class=\"label ");
+
+				if ((boolean)labelItem.get("dismissible")) {
+					jspWriter.write("label-dismissible ");
+				}
+
+				if ((boolean)labelItem.get("large")) {
+					jspWriter.write("label-lg ");
+				}
+
+				if (Validator.isNotNull(labelItem.get("displayType"))) {
+					jspWriter.write(
+						"label-" + labelItem.get("displayType") + " ");
+				}
+
+				jspWriter.write("\">");
+				jspWriter.write(String.valueOf(labelItem.get("label")));
+				jspWriter.write("</span>");
+			}
+
+			jspWriter.write("</div>");
+		}
+
+		jspWriter.write("</div>");
+
+		if (!ListUtil.isEmpty(_actionDropdownItems)) {
```

Comment:
@carloslancha aren't we missing a `<div class="autofit-col">` before the `<div class="dropdown">` (i.e. as a parent) ?

Reply:
Yup! wow, nice catch! Thx!!